### DISCUSSION TITLE
New addon discovery

### DISCUSSION THREAD

**Discussion by Illodrus** [ScratchAddons/ScratchAddons#991] [Ideas]
In the addons settings, could there be a "sort by new" or "sort by unseen" feature so that when new updates are released you can easily see what's new? Could be helpful for users who don't regularly check update schedules.

**Comment by WorldLanguages** [comment]
**I agree that we need to make it more obvious there are new addons (specially with the very requested onion skinning feature). Some ideas:**
1. Make the extension badge another color (like red). When the user clicks the icon, instead of opening the popup, it sends the user to the changelog (a bit more context might be needed)
2. Open the changelog (again, with more context than the current changelog, so that people know why it opened) for all users. Might be too much, since some users might have forgotten they have the extension.
3. Open the changelog (with context, see above) for anyone who has any editor or website addon enabled, that is not turned on by default.
4. Same as 3, but we check the user has accessed scratch.mit.edu recently. We could do this by looking at the session cookie registration dates (google analytics cookies might work as well, but some people might block them).
5. Link to the changelog when the user goes to scratch.mit.edu for the first time after the update - maybe we can add a pseudo notification in the bottom right that links to the changelog, and highlights the most important updates.
6. In the popup, show "protips" that remind the user of things they can do with certain addons. This could be combined with any of the above.

**Reply by WorldLanguages** [reply]
tagging for feedback: @apple502j @Hans5958 @GrahamSH-LLK @TheColaber

**Reply by GrahamSH-LLK** [reply]
Interesting. I think #1 is good. #6 is also good, but it can also be on the settings page.

**Reply by Hans5958** [reply]
Just an idea, just put a small banner on the pop-up indicating that SA is updated, and then remove it when it is seen. They can see it is updated when they wanted to look their messages (or opening the pop-up).

**Reply by WorldLanguages** [reply]
@Hans5958 I mean that's fine, but some users might not use the popup at all, but might heavily use the editor, for example. IMO we should do number 5 (which automatically excludes anyone who's not interested because they don't use Scratch anymore, but still includes people that don't use the popup). I guess some people might only use the popup and not the website, tho

**Reply by TheColaber** [reply]
I actually think I like the first option... I think It would make the most sense... plus it would be pretty easy to implement...

**Reply by Hans5958** [reply]
I think 5 is also appropriate considering RES also does the same IIRC
